.github/workflows/ci.yml

```diff
@@ -5,6 +5,8 @@ on:
     branches:
       - master
       - 'auto-cadence-upgrade/**'
+      - staging
+      - trying
```

Comment:
are we removing these later?

Reply:
No, these are needed by bors. It pushes to these branches to run CI. See [this](https://bors.tech/documentation/getting-started/#:~:text=The%20pull%20request%2C%20as,not%20be%20well-isolated.)